### DISCUSSION TITLE
[Kavita] Social reading with Kavita ?

### DISCUSSION THREAD

**Discussion by Maitresinh** [Kareadita/Kavita#2673] [Ideas]
### Idea Description

I wonder if there would be a way to use Kavita for Social Reading.
By Social reading, i mean reading with other people on the same book, sharing highlights, notes and messages on the book itself, like on the glose plateform

https://glose.com/what-is-glose

It's really a great new way to enhance the reading activity, that i enjoy  a lot, as a reader, but also as a non profit publisher and as a teacher.

As a publisher, it gives the possibly to organiez reading sessions with the author of translator for exemple. For students, obviously, groups or readers can help themselves along the process, and you can keep an eye on it as a teacher.

I would personnally pay for such a feature ( glose is fine, but you cannot upolad your books and share them. And the educational version is too expansive and restrictive).

### Duration of Using Kavita

one year, on a PI, and now my unraid nas

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by lyingloverr** [comment]
A nice idea, but so extremely niche.

**Comment by Maitresinh** [comment]
Well...crowdfunding was a niche totally idea 15 years ago. Before the progressive adoption.

I think the main obstacle for glose an others is their economic model. To get access to the social reading, you have to buy books from their own store.
